clarify subset in yss and ybld (#1034)
riscv/riscv-cheri: src/cheri/insns/cbld_32bit.adoc

. `{cs1}.tag=1`, and
. `{cs1}` passes all <<section_cap_integrity,integrity>> checks, and
. `{cs1}` is not sealed, and

> . `{cs2}` has no permission bit set in it's <<AP-field>> which is not set in `{cs1}` 's, and

Reviewer: Please don't talk about bits being set in encoding fields, as it constrains and complicates compressed encodings.  Instead, this should be "has no permission bit set in the architectural view of permissions as returned by `YPERMR` ..." or something along those lines.  Almost surely simpler to define the permission, bounds, and (so) capability subset relationships out of line and cite them here.

Author: ```suggestion
. `{cs2}` has no permission granted in its <<AP-field>> or <<SDP-field>> which is granted by `{cs1}`, and
```

Author: better? and yes it would be better to have a defn elsewhere. Let's agree the wording first.

Reviewer: Still no: SDP is not part of AP, and that also matters.  (Also "its", not "it's").

Author: better?

> . `{cs2}` has no permission granted in its <<AP-field>> or <<SDP-field>> which is granted by `{cs1}`, and

Reviewer: That seems to be missing a "not", but perhaps rephrasing with positive logic is better:

```suggestion
. all permissions granted by either the <<AP-field>> or <<SDP-field>> of `{cs2}` are also granted by those of `{cs1}`, and
```
